settings.gradle

```diff
@@ -114,15 +129,3 @@ if (localBuilds.exists()) {
         }
     }
 }
-
```

Comment:
If I'm not mistaken, I think we still need to configure the build cache – just pointing at Gradle Enterprise, not at the cache node.

Reply:
Yes, I think you are correct, it is just that atm I am not sure what we should be doing with `caching` in general, like disabling it completely, enable it completely, or replicate the setup we have on `caching` already so that we don't mess up with our current baseline, which is:

- Using the `org.gradle.caching=true` locally, and
- Using the `buildCache { ... }` on CI only, just pointing to `gradle.a8c.com` this time.

I recommend waiting on confirmation from the Gradle folks on that before proceeding with it. Wdyt?